Modules/_decimal/_decimal.c

```diff
@@ -1519,7 +1519,7 @@ init_extended_context(PyObject *v)
 #ifdef EXTRA_FUNCTIONALITY
 /* Factory function for creating IEEE interchange format contexts */
 static PyObject *
-ieee_context(PyObject *dummy UNUSED, PyObject *v)
+ieee_context(PyObject *self, PyObject *v)
```

Comment:
```suggestion
ieee_context(PyObject *module, PyObject *v)
```

Reply:
Actually, we wanted to keep the `self`. See https://github.com/python/cpython/pull/122082#discussion_r1720929120

Reply:
Yeah, that was unfortunate change :(

Reply:
@kumaraditya303, let me know if I should prepare a quick fix.

Reply:
It's fine, I would rather stick to module, see y reasoning above.